scripts/fetch-e2e.sh

```
@@ -35,9 +35,57 @@ BASE="http://127.0.0.1:${PORT}"
 run_js() {
   echo "$1" > "$TMPFILE"
-  "$LOADER" "$TMPFILE" --compat-asi 2>&1
+  "$LOADER" "$ALLOW_HOST" "$TMPFILE" --compat-asi 2>&1
+}
+
+# Same as run_js, but with extra loader flags before the script path, so the
+# policy flags can be exercised against the same live server.
+run_js_with() {
+  local extra="$1"
+  echo "$2" > "$TMPFILE"
+  # shellcheck disable=SC2086 -- $extra is a deliberate flag list
```

Shellcheck (0.11.0)

[error] Couldn't parse this shellcheck directive. Fix to allow more checks.

(SC1073)